components/03-modules/gallery/gallery.hbs

```diff
@@ -0,0 +1,86 @@
+<div class="
+         gallery {{ class }}
+         {{#if typeHorizontal }}
+             gallery--horizontal
+         {{else}}
+             gallery--vertical
+         {{/if}}
+     "
+     {{{ attributes }}}
+>
+    <div class="
```

Comment:
I think this elements here are for switching photos by i.e arrows in left and right side right? And clicking on it will switch image to next? But we don't have any styles for it

Reply:
I think you commented in wrong area but I removed arrows for main image as  don't see it in design.